konduit-serving-models/konduit-serving-tensorflow-python/src/main/java/ai/konduit/serving/models/tensorflowpython/NumpyArray.java

```diff
@@ -0,0 +1,105 @@
+package ai.konduit.serving.models.tensorflowpython;
+
+import ai.konduit.serving.pipeline.api.data.NDArrayType;
+import ai.konduit.serving.pipeline.api.format.NDArrayFactory;
+import ai.konduit.serving.pipeline.impl.data.ndarray.BaseNDArray;
+import org.nd4j.python4j.PythonGC;
+import org.nd4j.python4j.PythonObject;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class NumpyArray {
```

Comment:
It might make sense to put this in konduit-serving-python-new instead? There may be other uses for it.

Reply:
We already have INDArray based NumpyArray from python4j there.